### PR TITLE
Replace lodash utility with String native methods

### PR DESCRIPTION
**Pull request by exoego**
See https://github.com/serverless/serverless/pull/7680#discussion_r422044932

Let's minimize lodash dependency where native ways are available.

* Better performance thanks to native implementation
* Smaller bundle size thanks to depend only small portion of lodash (e.g. `require('lodash.pick')`)

### CODE REVIEW

File: lib/plugins/aws/common/lib/artifacts.js [hunk 1]
```diff
@@ -13,7 +12,7 @@ module.exports = {
       path.join(this.serverless.config.servicePath || '.', '.serverless');
 
     // Only move the artifacts if it was requested by the user
-    if (this.serverless.config.servicePath && !_.endsWith(packagePath, '.serverless')) {
+    if (this.serverless.config.servicePath && packagePath.endsWith('.serverless')) {
```
Comment (medikoo) [inline lib/plugins/aws/common/lib/artifacts.js:15]: If I see correctly negation was lost

File: lib/plugins/aws/common/lib/artifacts.js [hunk 2]
```diff
@@ -36,7 +35,7 @@ module.exports = {
...
```
Comment (medikoo) [inline lib/plugins/aws/common/lib/artifacts.js:38]: Same here

File: lib/plugins/aws/common/lib/artifacts.js (hunk 1 above)
Comment (exoego) [reply lib/plugins/aws/common/lib/artifacts.js]: Fixed and squashed commits.

File: lib/plugins/aws/common/lib/artifacts.js (hunk 2 above)
Comment (exoego) [reply lib/plugins/aws/common/lib/artifacts.js]: Fixed and squashed commits.

**Comment by medikoo** [comment]
@exoego is this ready to take? If so, please re-request review. Thank you!

**Comment by exoego** [comment]
Ready to re-review.

**Pull request merged**